app/src/main/java/com/picone/go4lunch/presentation/viewModels/RestaurantViewModel.java

```diff
@@ -71,8 +83,32 @@ public RestaurantViewModel(GetAllRestaurantsInteractor getAllRestaurantsInteract
 
     public LiveData<Restaurant> getSelectedRestaurant = selectedRestaurantMutableLiveData;
 
-    public List<Restaurant> getAllRestaurants() {
-        return getAllRestaurantsInteractor.getGeneratedRestaurants();
+    public LiveData<List<Restaurant>> getAllRestaurants = allRestaurantsMutableLiveData;
+
+    @SuppressWarnings("ResultOfMethodCallIgnored")
+    @SuppressLint("CheckResult")
```

Comment:
What is this for ?

Reply:
cause i need an observeValueEvent on the restaurants list to update interested users in ui RestaurantList and MapsMarker